components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -75,16 +75,8 @@ export default class CreateWorkspace extends React.Component<CreateWorkspaceProp
           </div>;
           break;
         case ErrorCodes.NOT_AUTHENTICATED:
-          const authorizeUrl = gitpodHostUrl.withApi({
-              pathname: '/authorize',
-              search: `returnTo=${encodeURIComponent(window.location.toString())}&host=${error.data.host}&scopes=${error.data.scopes.join(',')}`
-          }).toString();
-          window.location.href = authorizeUrl;
-          statusMessage = <div className="mt-2 flex flex-col space-y-8">
```

Comment:
Please undo this change 🙏🏻 

It disagrees with https://github.com/gitpod-io/gitpod/issues/3754. I'm about to add handling for another error mode: user needs to select an account.

Reply:
I think we can probably have both changes (faster redirect, and /select-account logic).

But I'm happy to wait for your change and rebase/re-implement mine on top. 🙂

Reply:
@jankeromnes no we cannot, because we need to open the extra tab instead of the redirect to the auth flow. that extra tab is executing the auth flow and is going to propagate the "select account" error.

Reply:
background: redirecting away from "this page" is always a risk of getting into a dead-end, so the user cannot find back.

Reply:
The `returnTo` URL seems to work generally well for that.

And, if there is indeed a rare/major complication in my auth flow, I think it's expected that I don't return to the exact same URL as before, right? (Because I need to deal with the bigger issue first, then later I can try again what I originally wanted to do.)

Reply:
But sure, I'll remove my change for now if it makes your more complicated.

Reply:
I've spliced my other commit into a separate branch: https://github.com/gitpod-io/gitpod/tree/jx/faster-start-auth-redirect

Next I'll wait for your `/select-account` change to re-evaluate.

Reply:
It’s not about the details of the flow, but it is integrated into the SPA after all. We agreed to try the open in new window with good reasons and knowing that it would introduce this extra step for the benefit of more control of the overall experience. 

Such an extra step is rare, that’s true. So the idea is to support user’s instead of assuming that they know how oauth works.

Also this PR doesn’t sound like it would do this 😉